File: 파이썬/7.py
score=input()
s1=0
s2=0
for i in range(len(score)):
    if i<len(score)//2:
        s1+=int(score[i]) 
    else :
        s2+=int(score[i])
if s1==s2:
    print("LUCKY")
else:
    print("READY")

File: 2024,2025/c++/7.cpp
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>
#include<iostream>
#include<algorithm>
int* vis;
using namespace std;
int main(void) {
    
   
    string str;
    int fsum = 0;
    int lsum = 0;
    cin >> str;
    for (int i = 0; i < str.length() / 2; i++) {
        fsum += str[i];
    }
    for (int i = str.length() / 2; i < str.length(); i++) {
        lsum += str[i];

    }
    if (fsum == lsum) {
        cout << "LUCKY";

    }
    else {
        cout << "READY";
    }
  
    
    


}